### ISSUE TITLE
RuntimeException: Unable to stop service org.wordpress.android.ui.uploads.UploadService@b2598bc: android.database.s...

### ISSUE THREAD

**Issue by AliSoftware** [wordpress-mobile/WordPress-Android#16107]
**Sentry Url**: https://sentry.io/share/issue/5622c5d6377f4b3984ae6a5dda28a679/
**User Count**: 4
**Count**: 1524
**First Release**: org.wordpress.android@17.5+1058
**First Seen**: 2021-06-18T10:43:55.747000Z
**Last Seen**: 2022-03-15T04:09:38Z
**24 Hours**: 16
**30 Days**: 319

```
RuntimeException: Unable to stop service org.wordpress.android.ui.uploads.UploadService@b2598bc: android.database.sqlite.SQLiteException: too many SQL variables (code 1 SQLITE_ERROR[1]): , while compiling: SELECT * FROM PostModel WHERE (_id IN (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,...
     at org.wordpress.android.fluxc.persistence.UploadSqlUtils.getPostModelsForPostUploadModels(UploadSqlUtils.java:178)
     at org.wordpress.android.fluxc.store.UploadStore.getAllRegisteredPosts(UploadStore.java:197)
     at org.wordpress.android.ui.uploads.UploadService.doFinalProcessingOfPosts(UploadService.java:998)
     at org.wordpress.android.ui.uploads.UploadService.onDestroy(UploadService.java:126)
...
(16 additional frame(s) were not displayed)
```

Not new… but 1.5k events (and growing)… so figured I'd still log it there.

**Comment by ravishanker** [comment]
This seems like a duplicate of #15078

**Comment by ravishanker** [comment]
Closed via #15078
